app/controllers/apo_controller.rb

```diff
@@ -34,18 +34,18 @@ def registration_options
     }.compact)
     @access_template = AccessTemplate.new(access_template:, apo_defaults_template: administrative.accessTemplate)
 
-    @collections = Array(administrative.collectionsForRegistration).filter_map do |col_id|
-      name = CollectionNameService.find(col_id)
-      unless name
-        Honeybadger.notify("The APO #{params[:id]} asserts that #{col_id} is a collection for registration, but we don't find that collection in solr")
+    @collections = Array(administrative.collectionsForRegistration).filter_map do |coll_id|
```

Comment:
Minor request: Get rid of the abbreviations in this method.

Reply:
I confess I was slightly confused by the original `col_id` because I read that as column, but `collection_id` is so much longer and the context of use is small here.

Reply:
I'm afraid I'm going to merge, and if you want to refactor, go for it.